### PR TITLE
completed issue-1-create-an-entry

### PR DESCRIPTION
**Pull request by anthony-urbina**
https://user-images.githubusercontent.com/109625982/189795894-7e4eca28-656f-43db-b3c3-f6e430e5e31c.mov

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+$photoURL.addEventListener('input', handleInput);
+
+function handleInput(event) {
+  var enteredURL = event.target.value;
+  $placeholderImg.setAttribute('src', enteredURL);
+}
+
+$form.addEventListener('submit', handleSubmit);
+
+var newEntry = {};
```
Comment (codingcodymiller) [inline js/main.js]: You should keep this inside the `handleSubmit` function, it doesn't need to be global.

File: js/main.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+function handleSubmit(event) {
+  newEntry = {
+    entryTitle: $title.value,
+    entryPhotoURL: $photoURL.value,
+    entryNotes: $notes.value,
+    entry: data.nextEntryId
+  };
+  data.nextEntryId++;
+  data.entries.unshift(newEntry);
+  event.preventDefault();
```
Comment (codingcodymiller) [inline js/main.js]: I would make this the first line of the function.

File: js/main.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  $placeholderImg.setAttribute('src', 'images/placeholder-image-square.jpg');
+  $form.reset();
+}
+
+window.addEventListener('beforeunload', handleUnload);
+
+function handleUnload(event) {
+  var dataJSON = JSON.stringify(data);
+  localStorage.setItem('data value', dataJSON);
+}
```
Comment (codingcodymiller) [inline js/main.js]: Please relocate this code to `data.js` so that it is in the same file as the variable it is preserving.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: js/main.js (hunk 3 above)
Comment (anthony-urbina) [reply js/main.js]: ahhhh nice idea

File: js/main.js (hunk 1 above)
Comment (anthony-urbina) [reply js/main.js]: got it

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**